### ISSUE TITLE
Reconstruction with Multi-Shell Multi-Tissue CSD

### ISSUE THREAD

**Issue by AbdallahKelani** [dipy/dipy#2190]
When following [the tutorial](https://dipy.org/documentation/1.1.1./examples_built/reconst_mcsd/) I get this error:

`ArpackNoConvergence: ARPACK error -1: No convergence (471 iterations, 0/1 eigenvectors converged)`

Which seems to be from the line:
`mcsd_fit = mcsd_model.fit(denoised_arr[:, :, 10:11])`

The warning preceding the error:
```
C:\fakepath\Anaconda3\envs\fakeenv\lib\site-packages\cvxpy\expressions\expression.py:516: UserWarning: 
This use of ``*`` has resulted in matrix multiplication.
Using ``*`` for matrix multiplication has been deprecated since CVXPY 1.1.
    Use ``*`` for matrix-scalar and vector-scalar multiplication.
    Use ``@`` for matrix-matrix and matrix-vector multiplication.
    Use ``multiply`` for elementwise multiplication.

  warnings.warn(__STAR_MATMUL_WARNING__, UserWarning)
```

-------

Also, from the denoising step I get this warning:

```
C:\fakepath\Anaconda3\envs\fakeenv\lib\site-packages\dipy\denoise\localpca.py:239: RuntimeWarning: invalid value encountered in true_divide
  denoised_arr = thetax / theta
```

Which I believe is from the line:
`denoised_arr = mppca(data, mask=mask, patch_radius=2)`

-------

I am using Anaconda on Windows. I have tried using Ubuntu as well and I seem to be getting the same error and warnings.

Any help you can provide is much appreciated.

**Comment by skoudoro** [comment]
Hi @AbdallahKelani,

Your first error is due to the new version of 'cvxpy' package so you can choose between this 2 temporary solutions:
- wait for DIPY release 1.3 
- downgrade 'cvxpy' version. It should be 1.0 or below.

Concerning your second error, I suppose this warning appears because of the background of your image so nothing to worry about. Just make sure to apply the right mask.

**Comment by arokem** [comment]
I wanted to flag that I am still seeing this: 

```
warnings.warn(__STAR_MATMUL_WARNING__, UserWarning)
/Users/arokem/.virtualenvs/afq/lib/python3.7/site-packages/cvxpy/expressions/expression.py:550: UserWarning: 
This use of ``*`` has resulted in matrix multiplication.
Using ``*`` for matrix multiplication has been deprecated since CVXPY 1.1.
    Use ``*`` for matrix-scalar and vector-scalar multiplication.
    Use ``@`` for matrix-matrix and matrix-vector multiplication.
    Use ``multiply`` for elementwise multiplication.
```

Even though I am running on current master.

**Comment by skoudoro** [comment]
It is not always easy to track. I think we are missing 2-3 updates for a specific case.

Can you provide the code/call which generates this warning @arokem?
